docs_zh-CN/tasks/2d_body_keypoint.md

```diff
@@ -1,3 +1,500 @@
+  - [MHP](#mhp) \[ [主页](https://lv-mhp.github.io/dataset) \]
+- 视频
+  - [PoseTrack18](#posetrack18) \[ [主页](https://posetrack.net/users/download.php) \]
+  - [sub-JHMDB](#sub-jhmdb-dataset) \[ [主页](http://jhmdb.is.tue.mpg.de/dataset) \]
+
+## COCO
+
+<!-- [DATASET] -->
+
+<details>
+<summary align="right"><a href="https://link.springer.com/chapter/10.1007/978-3-319-10602-1_48">COCO (ECCV'2014)</a></summary>
+
+```bibtex
+@inproceedings{lin2014microsoft,
+  title={Microsoft coco: Common objects in context},
+  author={Lin, Tsung-Yi and Maire, Michael and Belongie, Serge and Hays, James and Perona, Pietro and Ramanan, Deva and Doll{\'a}r, Piotr and Zitnick, C Lawrence},
+  booktitle={European conference on computer vision},
+  pages={740--755},
+  year={2014},
+  organization={Springer}
+}
+```
+
+</details>
+
+对 [COCO](http://cocodataset.org/) 数据集来说，请从此链接 [COCO download](http://cocodataset.org/#download) 下载， 2017 Train/Val 对于 COCO 关键点的训练和评估是非常必要的。
+[HRNet-Human-Pose-Estimation](https://github.com/HRNet/HRNet-Human-Pose-Estimation) 提供了 COCO val2017 的检测结果，来帮助重现我们的多人姿态估计的结果。
+请从 [OneDrive](https://1drv.ms/f/s!AhIXJn_J-blWzzDXoz5BeFl8sWM-) 或 [GoogleDrive](https://drive.google.com/drive/folders/1fRUDNUDxe9fjqcRZ2bnF_TKMlO0nB_dk?usp=sharing)下载。
+在 $MMPOSE/data 目录下下载并提取， 使得文件呈如下的格式：
+可选地, 为了在 COCO'2017 test-dev 上评估， 请下载 [image-info](https://download.openmmlab.com/mmpose/datasets/person_keypoints_test-dev-2017.json)。
+
+```text
+mmpose
+├── mmpose
+├── docs
+├── tests
+├── tools
+├── configs
+`── data
+    │── coco
+        │-- annotations
+        │   │-- person_keypoints_train2017.json
+        │   |-- person_keypoints_val2017.json
+        │   |-- person_keypoints_test-dev-2017.json
+        |-- person_detection_results
+        |   |-- COCO_val2017_detections_AP_H_56_person.json
+        |   |-- COCO_test-dev2017_detections_AP_H_609_person.json
+        │-- train2017
+        │   │-- 000000000009.jpg
+        │   │-- 000000000025.jpg
+        │   │-- 000000000030.jpg
+        │   │-- ...
+        `-- val2017
+            │-- 000000000139.jpg
+            │-- 000000000285.jpg
+            │-- 000000000632.jpg
+            │-- ...
+
+```
+
+## MPII
+
+<!-- [DATASET] -->
+
+<details>
+<summary align="right"><a href="http://openaccess.thecvf.com/content_cvpr_2014/html/Andriluka_2D_Human_Pose_2014_CVPR_paper.html">MPII (CVPR'2014)</a></summary>
+
+```bibtex
+@inproceedings{andriluka14cvpr,
+  author = {Mykhaylo Andriluka and Leonid Pishchulin and Peter Gehler and Schiele, Bernt},
+  title = {2D Human Pose Estimation: New Benchmark and State of the Art Analysis},
+  booktitle = {IEEE Conference on Computer Vision and Pattern Recognition (CVPR)},
+  year = {2014},
+  month = {June}
+}
+```
+
+</details>
+
+对于 [MPII](http://human-pose.mpi-inf.mpg.de/) 数据集, 请从此链接 [MPII Human Pose Dataset](http://human-pose.mpi-inf.mpg.de/) 下载。
+我们已经将原来的标注文件转成了json格式， 请从此链接 [mpii_annotations](https://download.openmmlab.com/mmpose/datasets/mpii_annotations.tar) 下载。
```

Comment:
space between cn and en

![image](https://user-images.githubusercontent.com/9464825/129871300-0d3dd0ab-ca01-4621-b931-214d158e4077.png)